debian/changelog

```diff
@@ -1,3 +1,14 @@
+rdma-core (12-1.1) UNRELEASED; urgency=medium
+
+    * New version.
+    * Adding debian/copyright.
+    * Fix debian/iwpmd.
+    * Fix debian/control.
+    * Initial Release.
+    * Close ITP (Closes: #848971).
+
+ -- Talat Batheesh <talatb@mellanox.com>  Mon, 26 Dec 2016 13:22:50 +0200
+
```

Comment:
Don't increment the Debian version before it has entered the official Debian archive!